test/common/network/dns_impl_test.cc

```diff
@@ -441,11 +441,12 @@ class DnsImplTest : public testing::TestWithParam<Address::IpVersion> {
         Network::Test::getCanonicalLoopbackAddress(GetParam()));
     listener_ = dispatcher_->createListener(socket_, *server_, true);
     updateDnsResolverOptions();
+    envoy::config::core::v3::DnsResolverOptions dns_resolver_options = dns_resolver_options_;
```

Comment:
Can you add a comment here explaining what we're testing by putting this on the stack?

Reply:
makes sense.